src/main/resources/admin/tools/content-type-manager/components/structure/component-field-set.html

```diff
@@ -71,8 +71,9 @@
           return `${this.path}elements[${index}]`;
         },
 
-        removeSet() {
-          this.$store.commit("removeSchemaInJson", { path: this.path });
+        async removeSet() {
+          if (await app.$root.confirmDialog("Are you sure ?"))
+            this.$store.commit("removeSchemaInJson", { path: this.path });
```

Comment:
This might just me being picky, but could you add "{ }" brackets to if statments? 
Single line if statments i almost never see in any code.